src/Input/Keyboard.cs

```diff
@@ -25,9 +25,21 @@ public static class Keyboard
 		/// Returns the current keyboard state.
 		/// </summary>
 		/// <returns>Current keyboard state.</returns>
-		public static KeyboardState GetState()
+		public unsafe static KeyboardState GetState()
 		{
-			return new KeyboardState(keys);
+			// Returns a pointer that does not need to be freed.
```

Comment:
We should inline this into `FNAPlatform.GetKeyboardState(List<Keys>)`, so that each `i` doesn't have to call the delegate in a tight loop. This also makes it possible to move the ControlDown bools to SDL2_FNAPlatform.